src/Microsoft.CorrelationVector/CorrelationVectorExtensions.cs

```diff
@@ -34,7 +34,7 @@ public static Guid GetBaseAsGuid(this CorrelationVector correlationVector)
                 throw new InvalidOperationException("Cannot convert a V1 correlation vector base to a guid.");
             }
 
-            if (CorrelationVector.ValidateCorrelationVectorDuringCreation)
+            if (CorrelationVectorV2.ValidateCorrelationVectorDuringCreation)
```

Comment:
We should use CorrelationVector.ValidateCorrelationVectorDuringCreation everywhere, not the one with the version